src/entities/BlockTune/index.ts

```diff
@@ -16,23 +15,36 @@ export class BlockTune {
    */
   #data: Record<string, unknown>;
```

Comment:
maybe generic should be used here instead of `Record<string, unknown>`?

Reply:
Good question. I think we might use a generic type here. But for now, I'm not sure why we need to know the type of this data object in our document model. So we always can change it, but in a separate pr when we'll need it.